script/DeploySpark.s.sol

```diff
@@ -70,52 +70,96 @@ contract DeploySpark is Script {
     WrappedTokenGatewayV3 wethGateway;
     WalletBalanceProvider walletBalanceProvider;
 
+    Pool poolImpl;
+
+    InitializableAdminUpgradeabilityProxy incentivesProxy;
+    RewardsController rewardsController;
+    IEACAggregatorProxy proxy;
+
     function run() external {
         //vm.createSelectFork(vm.envString("ETH_RPC_URL"));     // Multi-chain not supported in Foundry yet (use CLI arg for now)
         instanceId = vm.envOr("INSTANCE_ID", string("primary"));
         vm.setEnv("FOUNDRY_ROOT_CHAINID", vm.toString(block.chainid));
 
         config = ScriptTools.loadConfig("config");
 
-        admin = config.readAddress(".admin");
+        admin    = config.readAddress(".admin");
         deployer = msg.sender;
 
         vm.startBroadcast();
-        registry = new PoolAddressesProviderRegistry(deployer);
+
+        // 1. Deploy and configure registry and addresses provider
+
+        registry              = new PoolAddressesProviderRegistry(deployer);
         poolAddressesProvider = new PoolAddressesProvider(config.readString(".marketId"), deployer);
+
         poolAddressesProvider.setACLAdmin(deployer);
+
+        // 2. Deploy and data provider and pool configurator
```

Comment:
"Deploy and configure data provider..." missing word. Also, there is no mention of the initialization.

Reply:
Updated